src/main/resources/application.yml

```diff
@@ -68,6 +62,12 @@ workspace:
     frequency-ms: 60000
     staleness-threshold-ms: 600000
 
+  tracing:
+    enabled: ${env.tracing.enabled}
+    probability: ${env.tracing.probability}
+    project-id: ${env.tracing.project-id}
+    sa-path: ${env.tracing.sa-path}
```

Comment:
not really a comment on this PR, but why split these into the `env` declaration with shadow environment variables and the usage here? When should I have a `env` variable, and should I ever override them in other application property files.
We can override the values here from environment variables directly already, e.g. Spring maps `WORKSPACE_TRACING_ENABLED` to the workspace.tracing.enabled proprety. We don't need `CLOUD_TRACE_ENABLED` as well.